interfaces/builtin/modem_manager.go

```diff
@@ -199,6 +202,9 @@ dbus (send)
     path=/org/freedesktop/ModemManager1{,/**}
     interface=org.freedesktop.DBus.Properties
     member="Get{,All}",
+
+# allow communicating with mbim-proxy server
+unix (connect, receive, send) type=stream peer=(addr="@{mbim,qmi}-proxy"),
```

Comment:
Are you sure these are needed here too? It seems to me that the `modemManagerConnectedPlugAppArmorClassic` snippet is used *in addition* to the `modemManagerConnectedPlugAppArmor` snippet, so only the latter should be needed.

Reply:
Let me test it later, thanks.

Reply:
Yes, the part in **modemManagerConnectedPlugAppArmorClassic** is NOT needed.